Client/Frontend/Browser/URLBarView.swift

```diff
@@ -269,6 +271,13 @@ class URLBarView: UIView {
             make.trailing.equalTo(self)
             make.width.height.equalTo(UIConstants.ToolbarHeight)
         }
+
+        // Add an offset to the left for slide animation, and a bit of extra offset for spring bounces
+        let leftOffset: CGFloat = self.tabsButton.frame.width + URLBarViewUX.URLBarCurveOffset + URLBarViewUX.URLBarCurveBounceBuffer
```

Comment:
I think I got rid of this offset in one of my latest patches since I updated the constraints to be along the sides. Do we need this anymore?